Makefile

```diff
@@ -67,22 +67,25 @@ html:
 	mkdir -p $(BUILD_DIR) && cp -r src/html/ $(BUILD_DIR)
 
 # TODO: find a nice way to keep this DRY and dynamic
-cache/packages-017:
+cache/packages-017: cache/
```

Comment:
This should probably be a prerequisite for the `download` rule, so there is no duplication.